enterprise/backend/test/metabase_enterprise/transforms/ordering_test.clj

```diff
@@ -46,36 +50,131 @@
   (mt/with-metadata-provider (mt/id)
     (#'ordering/transform-deps transform)))
 
+(deftest not-run-transform-dependency-ordering-test
+  (mt/test-driver :postgres
```

Comment:
Could we run this for more drivers?

Reply:
If HoneySQL cannot solve/alleviate this problem, then leave it as is. It's probable not worth a lot of effort.